storage/remote/otlptranslator/prometheusremotewrite/number_data_points.go

```diff
@@ -72,7 +72,7 @@ func (c *PrometheusConverter) addGaugeNumberDataPoints(ctx context.Context, data
 }
 
 func (c *PrometheusConverter) addSumNumberDataPoints(ctx context.Context, dataPoints pmetric.NumberDataPointSlice,
-	resource pcommon.Resource, metric pmetric.Metric, settings Settings, metadata prompb.MetricMetadata, scope scope,
+	resource pcommon.Resource, _ pmetric.Metric, settings Settings, metadata prompb.MetricMetadata, scope scope,
```

Comment:
I was in doubt if we wanted to remove the argument or just keep it there for backwards compatibility 🤔

Reply:
Just remove the unused argument please. It doesn't matter in any case, since it's just an unexported method (not public API).